src/file_readers.py

```diff
@@ -0,0 +1,123 @@
+"""
+Notes
+-----
+This module contains the functions for audiobook_gen that read in the
+file formats that require for parsing than plain text (pdf, html, epub),
+as well as the preprocessing function for all input files.
+"""
+
+__all__ = ['preprocess', 'read_pdf', 'read_epub']
```

Comment:
You don't need a `__all__` (pronounced "dunder all") here or in any of your modules since you're already using all these functions. The `__all__` is only for when you want to hide functionality (i.e. functions, classes, variables) in the file from a user importing them. In your case, you're the only one importing them and using them (in your app) so it doesn't matter. This is more relevant for python packages, which yours is not.

Reply:
Understood - I've removed the `__all__` from the source files.